msystems/migrations/0003_add_search_policyholders_perms.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 3.2.21 on 2023-11-03 16:46
+
+from django.db import migrations
+
+from core.models import Role, RoleRight
```

Comment:
Please change model imports to apps.get_models

Reply:
You are right. I've changed that.